### ISSUE TITLE
Fractions for sleep & integers for amplitude

### ISSUE THREAD

**Issue by tjvr** [cambridge-alpha-team/unvisual-frontend#51]
We still need to display/announce sleep values as _fractions_, as described in #8.

**Comment by isaac-dunn** [comment]
There are other values that are also incremented by 0.125, and unless you realise that 0.125 is an eighth, it might be confusing why the first option for e.g. amplitude is 0.125.

Perhaps we could use fractions for those things too, or even use 1, 2, ..., 8 instead of 0.125, 0.25, ..., 1.0, to avoid confusion?

**Comment by tjvr** [comment]
> There are other values that are also incremented by 0.125

I'm only suggesting using fractions for sleep, since it makes sense musically to express durations that way.

Hmm. I don't see why amplitudes should use eighths; a step of 0.1 would probably be better. Or we could borrow a design choice from Scratch, and use the range `0`–`100` instead of `0.0`–`1.0`. I assume Scratch has good reasons for doing it that way: maybe percentages are more intuitive than decimals.  (I think this is similar to your last suggestion above.)

**Comment by isaac-dunn** [comment]
Yeah I'm basically saying that increasing amplitude (and I think there's others too) in eighths is definitely non-intuitive. Using `0.1` to `1.0` would certainly be an improvement, but would integers be even better?

**Comment by RusticFlare** [comment]
I really like the idea of changing amplitude to a range of 1-10.

According to http://education.more4kids.info/46/homeschooling-math-concepts-and-skills-by-age/ children over 8 are expected to learn fractions, decimals and basic percentages. But I still think using integers are much more intuitive.

**Comment by isaac-dunn** [comment]
Cool, let's go with natural numbers in those situations then.

**Comment by KoviRobi** [comment]
Also, for fractions, I think ±1/2 corresponds to a sharp/flat in music, so we might also want to have fractions of halves.

**Comment by tjvr** [comment]
> I think ±1/2 corresponds to a sharp/flat in music

What? We're talking about sleep durations here! I believe Sonic Pi uses MIDI notes for `play`, so we don't need to add ½-increments for semitones—but regardless, that would be a separate issue :-)
